src/client/startup.js

```diff
@@ -147,6 +147,8 @@ sand.globalFunctions = {
 					}
 				}
 			});
+			// running this after the ajax request in case the user has analytics blocked
```

Comment:
i don't understand this comment -- are you afraid that ```ga()``` might be undefined, causing the script to error out?